platform/spire/resources/setup.yaml

```diff
@@ -5,6 +5,7 @@ cluster:
   kerberos-realm: ATHENA.MIT.EDU
   mirror: debian.csail.mit.edu/debian
   user-grant-domain: homeworld.mit.edu
+  user-grant-email-domain: MIT.EDU
```

Comment:
There's one more copy of `setup.yaml` in the `deploy-chroot` directory (should be moved, but that's where it is right now).  Can you update that one too?